### ISSUE TITLE
Using Shift as a meta key in Keyboard Shortcuts.

### ISSUE THREAD

**Issue by raguay** [fman-community/fman#181]
I'm only able to get the Shift key to work as a meta key in shortcuts with alphabetical keys. For example, setting a command to "Shift+." will not work. Single letter commands also don't work, but I'm not sure if that is necessary.

**Comment by mherrmann** [comment]
I would assume "Shift+." gets interpreted as ":". Have you tried that?

**Comment by raguay** [comment]
Yes. ">" isn't assignable as a hotkey.

**Comment by alphaniner** [comment]
> 
> 
> I would assume "Shift+." gets interpreted as ":". Have you tried that?

@mherrmann Could you explain further? Does this mean that pressing "Shift+." should activate a keybinding for ":"? Or vice versa?

In any case I'm trying to assign ':' and '?' and presumably running into the same issue (the bindings are ignored).

**Comment by mherrmann** [comment]
@alphaniner I just debugged this. fman uses the [Qt framework](https://www.qt.io) to draw its GUI. This framework handles the key processing. Unfortunately, in the cases of `:` and `?`, it delivers `Shift+:` and `Shift+?`, respectively. You'll have to use these "keys" to map `:` and `?`. Sorry.

**Comment by alphaniner** [comment]
No need to apologize. It's unintuitive but I suppose there's nothing to be done. I imagine the framework maintainers are aware, and either don't consider it a bug or maintain it for compatibility.

That said, maybe you could mention the workaround on the [custom shortcuts page](https://fman.io/docs/custom-shortcuts) for the benefit of others.

Regardless, thanks!

**Comment by mherrmann** [comment]
Good idea. I've updated the [docs](https://fman.io/docs/custom-shortcuts), like you suggested.
